source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -54,9 +54,10 @@ these tests::
   _mongodb._tcp.test18.test.build.10gen.cc.   86400  IN SRV  27017  localhost.sub.test.build.10gen.cc.
   _mongodb._tcp.test19.test.build.10gen.cc.   86400  IN SRV  27017  localhost.evil.build.10gen.cc.
   _mongodb._tcp.test19.test.build.10gen.cc.   86400  IN SRV  27017  localhost.test.build.10gen.cc.
-  _mongodb._tcp.test20.test.build.10gen.cc.   86400  IN SRV  27017  localhost.test.build.10gen.cc.
+  _mongodb._tcp.test20.test.build.10gen.cc.   86400  IN SRV  8000   localhost.test.build.10gen.cc.
   _mongodb._tcp.test21.test.build.10gen.cc.   86400  IN SRV  27017  localhost.test.build.10gen.cc.
-  _customname._tcp.test22.test.build.10gen.cc 86400  IN SRV  27017  localhost.test.build.10gen.cc
+  _customname._tcp.test22.test.build.10gen.cc 86400  IN SRV  27017  localhost.test.build.10gen.cc.
+  _mongodb._tcp.test23.test.build.10gen.cc.   86400  IN SRV  8000   localhost.test.build.10gen.cc.
```

Comment:
Just want to clarify my understanding. From offline conversation, it sounds like load balancer tests make use of the `test1`, `test3`, `test4` and `test20` records. The `test1` and `test4` records seem to be used to test that drivers fail during validation, so updating the port from 27017 (the `mongos`) to 8000 (the actual LB) is not necessary. The `test3` record is used to test a successful connection, so the port _should_ be updated, but that record is also used in the [`load-balanced/srvMaxHosts-zero`](https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/tests/load-balanced/srvMaxHosts-zero.yml#L2) test which expects to find 27017. So, we need a new `test23` record that is identical to `test3` but points to port 8000. Let me know if that all sounds right.

Reply:
That's correct (the `test3` record is currently used in various `replica-set` tests, but you're right that it's required to stay the same for other tests).

Reply:
Ok, update: This change is now strictly additive. There are two new records, `test23` (replaces `test3`) and `test24` (replaces `test20`).